BoxContentSDK/BoxContentSDK/Requests/BOXRecentItemsRequest.h

```diff
@@ -0,0 +1,41 @@
+//
+//  BOXRecentItemsRequest.h
+//  Pods
+//
+//  Created by Andrew Dempsey on 12/19/16.
+//
+//
+
+#import <BoxContentSDK/BoxContentSDK.h>
+
+@interface BOXRecentItemsRequest : BOXRequest
+
+/**
+ Number of records to fetch. 
+ - Minimum allowed value: 0
+ - Maximum allowed value: 1000
+ - Defaults to 100
+ */
+@property (nonatomic, readwrite, assign) NSInteger limit;
+
+/**
+ Marker of the recents record to start from
+ */
+@property (nonatomic, readwrite, strong) NSString *nextMarker;
+
+/**
+ Selects which filter should be applied to the recent items list.
+ Possible values are:
+ - BOXAPIRecentItemsTypeShared
+ - Defaults to no filter
+ */
```

Comment:
Should explain what the outcome is. This means that you can request only the subset of recents which were accessed via a shared link (vs all)?